### PR TITLE
Print log message with exact image & tag when pushes are complete

### PR DESCRIPTION
**Pull request by dmcwhorter-ddl**

### CODE REVIEW

File: domino-build.sh [hunk 1]
```diff
@@ -47,10 +47,14 @@ if [ "${nopush_flag}" == "" ]; then
...
+  docker push "${operator_target}"
+  echo -e "\n  *** Pushed operator to ${operator_target} *** \n"
 
+  executor_target="quay.io/domino/seldon-core-executor:${TARGET_IMAGE_TAG}"
   echo -e "\n  Pushing executor...\n"
-  docker push "quay.io/domino/seldon-core-executor:${TARGET_IMAGE_TAG}"
+  docker push "${executor_target}"
+  echo -e "\n  *** Pushed executor to ${executor_target} *** \n"
 
 fi
```
Comment (pocheung1) [inline domino-build.sh]: nit: missing newline

**Review by pocheung1 — APPROVED** [review]

File: domino-build.sh (hunk 1 above)
Comment (dmcwhorter-ddl) [reply domino-build.sh]: will add this, not sure how it is missing

Comment (fernandoacorreia) [reply domino-build.sh]: make sure your editor is configured to always add a newline.

Comment (dmcwhorter-ddl) [reply domino-build.sh]: Yeah, IDEA has always done this so I'm not sure why it wouldn't be doing it now.  But I did find a setting ticked off in the remote jetbrains client so maybe its a remote IDEA issue.

**Pull request merged**